specs/http-api/README.md

```diff
@@ -345,7 +350,43 @@ Uses DID authn via Bearer token in header.
 | ----- | ------------------------ |
 | id    | exchange id(s) to return |
 
+---
+
+## List Balances
+This endpoint is *OPTIONAL*. It is only relevant for PFIs which expose a stored balance functionality either for institutional or retail customers.
+
+### Description
+Returns an array of balance amounts for each currency held by the caller.  
+
+### Authentication
+Uses DID authn via Bearer token in header.
```

Comment:
We've got this `protected` language written for other endpoints as a shorthand for bearer tokens
```suggestion
### Protected
True
```